src/core/foot/foot.view.d.ts

```diff
@@ -0,0 +1,58 @@
+import {View} from '../view/view';
+import * as columnService from '../column/column.service';
+import {Aggregation} from '../services';
+import {Log, AppError} from '../infrastructure';
+import {getFactory as valueFactory} from '../services/value';
+import {Table} from "../dom/table";
+import {ColumnModel} from "../column-type/column.model";
+
+export declare class FootView extends View {
+	constructor(public table: Table);
+
+	rows: any[];
+	columns: ColumnModel[];
+	valueFactory: valueFactory;
+	model.viewChanged.watch(() => this.invalidate(model));
```

Comment:
hm, looks that this file contains copy-pasted logic from js file, probably it haven't finished yet, please remove :)